lib/csv_row_model/import/attributes.rb

```diff
@@ -38,6 +38,7 @@ def original_attribute(column_name)
         return nil unless csv_string_model.errors[column_name].blank?
 
         value = self.class.format_cell(mapped_row[column_name], column_name, self.class.index(column_name), context)
+        value = value.to_s if value.is_a? FalseClass
```

Comment:
` s =~ BooleanFormatValidator::FALSE_BOOLEAN_REGEX` won't work correctly if value is `false` instead of `'false'`. So I decided to just check for `FalseClass` and change to string